controller/integration/test/multifinalizer/multi_finalizer_test.go

```diff
@@ -0,0 +1,255 @@
+// +build k8srequired
+
+package multifinalizer
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger/microloggertest"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/controller/integration/testresource"
+	"github.com/giantswarm/operatorkit/controller/integration/wrapper/configmap"
+)
+
+const (
+	objName       = "test-obj"
+	testNamespace = "integration-parallel-test"
+
+	controllerNameA = "test-controller-a"
+	resourceNameA   = "test-resource-a"
+	testFinalizerA  = "operatorkit.giantswarm.io/test-controller-a"
+
+	controllerNameB = "test-controller-b"
+	resourceNameB   = "test-resource-b"
+	testFinalizerB  = "operatorkit.giantswarm.io/test-controller-b"
+)
+
```

Comment:
I think it would be good to have godoc with a summary of the test. Like we have in e2etests. Makes it easier to quickly understand what the test is about.

https://github.com/giantswarm/e2etests/blob/c0ce5912a4d7bee5923c664cd0c6ac6960a069f4/scaling/spec.go